tracer/build/_build/Build.Steps.cs

```diff
@@ -552,6 +552,73 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
             }
         });
 
+    Target DownloadLibDatadog => _ => _
+                .Unlisted()
+                .After(CreateRequiredDirectories)
+                .Executes(async () =>
+                {
+                    if (IsLinux || IsOsx)
+                    {
+                        if (!Directory.Exists(NativeBuildDirectory))
+                        {
+                            CMake.Value(
+                                arguments: $"-DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -B {NativeBuildDirectory} -S {RootDirectory}");
+                        }
+                        // In CI, the folder might already exist. Which means that CMake was already configured
+                        // and libdatadog artifact (correct one) was already downloaded.
+                        // So just reuse it.
+
+                    }
+                    else if (IsWin)
+                    {
+                        var vcpkgExePath = await GetVcpkg();
+                        var vcpkgRoot = Environment.GetEnvironmentVariable("VCPKG_ROOT") ?? Directory.GetParent(vcpkgExePath).FullName;
+                        var vcpkg = ToolResolver.GetLocalTool(vcpkgExePath);
+                        foreach (var arch in new[] { MSBuildTargetPlatform.x64, MSBuildTargetPlatform.x86 })
+                        {
+                            // This big line is the same generated by VS when installing libdatadog while building the profiler
+                            vcpkg($"install --x-wait-for-lock --triplet \"{arch}-windows\" --vcpkg-root \"{vcpkgRoot}\" \"--x-manifest-root={RootDirectory}\" \"--x-install-root={BuildArtifactsDirectory}\\deps\\vcpkg\\{arch}-windows\" --downloads-root {BuildArtifactsDirectory}\\obj\\vcpkg\\downloads --x-packages-root {BuildArtifactsDirectory}\\obj\\vcpkg\\packages --x-buildtrees-root {BuildArtifactsDirectory}\\obj\\vcpkg/buildtrees  --clean-after-build");
+                        }
+                    }
+                });
+
+    Target CopyLibDatadog => _ => _
+                .Unlisted()
+                .After(DownloadLibDatadog)
+                .Executes(() =>
+                {
+                    if (IsWin)
+                    {
+                        var vcpkgDepsFolder = BuildArtifactsDirectory / "deps" / "vcpkg";
+                        foreach (var architecture in new[] { MSBuildTargetPlatform.x64, MSBuildTargetPlatform.x86 })
+                        {
+                            var source = vcpkgDepsFolder / $"{architecture}-windows" / $"{architecture}-windows";
+                            if (BuildConfiguration == Configuration.Debug)
+                            {
+                                source /= "debug";
+                            }
+
+                            var dllFile = source / "bin" / "datadog_profiling_ffi.dll";
```

Comment:
is there a reason we are keeping the library name different between win vs unix/osx?

otherwise we have to deal with naming at interop level.

Reply:
The name should be change from libdatadog repository.
I worked on it (using the new builder) but on the windows it fails at building the binary due to a link error